.editorconfig

```diff
@@ -149,8 +149,7 @@ dotnet_public_api_analyzer.require_api_files = true
 
 # IDE0055: Fix formatting
 # Workaround for https://github.com/dotnet/roslyn/issues/70570
```

Comment:
This comment should also be removed

Reply:
This cannot be removed yet as #70570 is fixed in 17.9, which is not yet part of the compiler toolset used by the repo.

Reply:
The comment mentions that the following line is a workaround, but sets a severity to warning like we would expect for the non-workaround case. Can you clarify what the workaround is?

Reply:
Workaround is to specify this entry in editorconfig instead of globalconfig, which is #70570